src/components/section.js

```diff
@@ -1,19 +1,23 @@
 import React from "react";
-import { Grid, Button } from "@trussworks/react-uswds";
+import { Grid, GridContainer, Button } from "@trussworks/react-uswds";
 
 function Section({ section, buttons }) {
   return (
-    <Grid row className="usa-section">
-      <Grid>
-        {section?.heading && (
-          <h2 className="font-heading-xl">{section.heading}</h2>
-        )}
-        {section?.text && <p className="usa-intro">{section.text}</p>}
-        {buttons?.callToAction && (
-          <Button className="usa-button--big">{buttons.callToAction}</Button>
-        )}
+    <GridContainer className="usa-section">
+      <Grid row>
+        <Grid>
+          {section?.heading && (
+            <h2 className="font-heading-xl">{section.heading}</h2>
+          )}
+          {section?.text && <p className="usa-intro">{section.text}</p>}
+          {buttons?.callToAction && (
+            <Button className="usa-button usa-button--big">
```

Comment:
Is it necessary to assign class `usa-button` if the component is taken from `react-uswds`?

Reply:
Yeah because I did custom overrides on the usa-button class

Reply:
Yes, but I believe `Button` is replaced with `<button class="usa-button"></button>` when it is rendered. Try removing the class name and see if your overrides are still there.

Reply:
Yeah seems like the class name isn't needed! I just removed it